### DISCUSSION TITLE
Lost Access to GitHub Copilot for Student

### DISCUSSION THREAD

**Discussion by DXOGO** [community/community#59352] [Copilot Conversations]
### Select Topic Area

General

### Body

Hello everyone.

I lost my access to GitHub Copilot about a week or so ago. I went to https://education.github.com/discount_requests/application and applied for a new student program, which was approves as of 22 June 2023. However, I still can't get access to GitHub Copilot. What might be the problem?

Any advice or direction would be deeply appreciated. Thank you in advance for your time and help.

**Comment by davecheney** [comment]
@DXOGO you need to apply the coupon you received in your student developer pack to your account, then you can go to copilot.github.com, sign up, and the option for free student access for two years will be available.

**Reply by DXOGO** [reply]
What do you mean a coupon?

**Reply by davecheney** [reply]
Your student developer pack includes a coupon code which can be applied to your account to receive free copilot student access for two years.

**Reply by DXOGO** [reply]
Hello, thank you for answering me.

Unfortunately I don't recall getting any coupon code and as I'm aware my account still only has access to GitHub Free, even though in https://education.github.com/discount_requests/application GitHub says that I do not need to re-verify as a student because the last time I did that (June 22, 2023) it was approved so at least I should be able to have access to GitHub Pro (which I do not as of now). 

Really don't know which step to take next, I would appreciate some guidance on this.

**Reply by davecheney** [reply]
Thank you. Your best path to resolution is to contact support.
